### PR TITLE
Cleanup for the Unit tests

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

### CODE REVIEW

File: gluster-object/CentOS/docker-gluster-swift/Dockerfile [hunk 1]
```diff
@@ -9,6 +9,17 @@ MAINTAINER Prashanth Pai <ppai@redhat.com>
 # Install gluster-swift dependencies. To be removed when RPMs become available.
 # Clean downloaded packages and index
 
+LABEL architecture="x86_64" \
+      name="gluster/gluster-swift" \
+      release="1" \
+      version="3.10" \
+      vendor="Red Hat, Inc" \
+      summary="This image has a running glusterfs swift service ( CentOS 7 + Gluster swift 3.10)" \
```
Comment (prashanthpai) [inline gluster-object/CentOS/docker-gluster-swift/Dockerfile]: "This image has gluster-swift service running"

File: gluster-object/CentOS/docker-gluster-swift/Dockerfile [hunk 2]
```diff
@@ -9,6 +9,17 @@ MAINTAINER Prashanth Pai <ppai@redhat.com>
...
+      io.k8s.display-name="Gluster swift 3.10 based on CentOS 7" \
+      io.k8s.description="Gluster Swift Image is based on CentOS Image which enables files and directories created on GlusterFS to be accessed as objects via the Swift API." \
+      description="Gluster Swift Image is based on CentOS Image which enables files and directories created on GlusterFS to be accessed as objects via the Swift API." \
```
Comment (prashanthpai) [inline gluster-object/CentOS/docker-gluster-swift/Dockerfile]: "objects via the Swift and S3 API"

**Comment by MohamedAshiqrh** [comment]
Addressed the comments.

**Comment by humblec** [comment]
LGTM

**Pull request merged**